errbot/backends/irc.py

```diff
@@ -289,6 +291,13 @@ def connect(self, *args, **kwargs):
 
     def on_welcome(self, _, e):
         log.info("IRC welcome %s" % e)
+
+        # try to identify with NickServ if there is a NickServ password in the
+        # config
+        if self.nickserv_password:
+            msg = 'identify %s' % self.nickserv_password
```

Comment:
@gbin note that I am only using this convention because I found it in other places in the codebase. Under normal circumstances I would use `.format` here and I am happy to change this accordingly.